shared/config.gradle

```diff
@@ -46,6 +46,13 @@ nativeUtils.platformConfigs.named(nativeUtils.wpi.platforms.windowsx64).configur
     it.cppCompiler.args.add("/D_DISABLE_CONSTEXPR_MUTEX_CONSTRUCTOR")
 }
 
+// Use C++23 on Windows
+nativeUtils.platformConfigs.each {
+  if (it.name.contains('windows')) {
+    it.cppCompiler.args.add("/std:c++latest")
```

Comment:
Hmm. This might be a problem. I'm not sure if mixing /std:c++latest and /std:c++20 is safe.

Reply:
From what I've read, `/std:c++latest` isn't guaranteed to have stable standard support. MSVC doesn't have a `/std:c++23` flag yet.

Reply:
Let me see if theres a way in gradle to just set that one file to c++latest, at least for the gradle build. I'd prefer limiting the scope as much as possible.

Reply:
Does it need to be set to c++latest to use std::stacktrace?

Reply:
Yes. https://godbolt.org/z/W4WWrc9qG